bin/ghe-backup

```diff
@@ -8,8 +8,8 @@
 set -e
 
 # Bring in the backup configuration
-cd $(dirname "$0")/..
-. share/github-backup-utils/ghe-backup-config
+export GHE_BACKUP_ROOT="$( cd "$( dirname "${BASH_SOURCE[0]}" )/.." && pwd )"
```

Comment:
We should export from `ghe-backup-config` only, and use `. $( dirname "${BASH_SOURCE[0]}" )/ghe-backup-config` to source the `ghe-backup-config` script (or whatever path is required if the script is not in `share/backup-utils`). That'll prevent globals pollution an indadvertedly changing that global value.

We need to pay attention to `.deb` installs here and make sure the changes work with that too. If `GHE_BACKUP_ROOT` points to `/usr` in that case, we should be fine. We'll need to make sure the config search path does not change though.